### PR TITLE
feat(db): maintenance sync hostname information

### PR DESCRIPTION
**Pull request by vinayg-intel**
## PR Checklist

- [ ] Unit Tests have been added for new changes
- [ ] API tests have been updated if applicable
- [ ] All commented code has been removed
- [ ] If you've added a dependency, you've ensured license is compatible with Apache 2.0 and clearly outlined the added dependency.

### CODE REVIEW

File: data/initMPS.sql [hunk 1]
```diff
@@ -14,6 +14,8 @@ CREATE TABLE IF NOT EXISTS devices(
       connectionstatus boolean,
       mpsusername text,
       tenantid varchar(36),
+      friendlyhostname varchar(256),
```
Comment (rsdmike) [inline data/initMPS.sql:17]: rename to `friendlyname`, use camlCase for  column rename in the query

File: data/initMPS.sql [hunk 2]
```diff
@@ -14,6 +14,8 @@ CREATE TABLE IF NOT EXISTS devices(
...
+      friendlydnssuffix varchar(256),
```
Comment (rsdmike) [inline data/initMPS.sql:18]: rename to `dnssuffix`, use camlCase for  column rename in the query

File: src/data/postgres/tables/device.test.ts [hunk 3]
```diff
@@ -57,6 +57,8 @@ describe('device tests', () => {
       connectionstatus as "connectionStatus",
       mpsusername as "mpsusername",
       tenantid as "tenantId"
+      friendlyhostname as "friendlyhostname"
```
Comment (rsdmike) [inline src/data/postgres/tables/device.test.ts:60]: these tests don't look correct - they are missing commas between the columns in the query

**Review by rsdmike — CHANGES_REQUESTED** [review]

File: data/initMPS.sql (hunk 2 above)
Comment (vinayg-intel) [reply data/initMPS.sql]: Hi Mike. I googled and see that usually database entries to be in small case. We can have a sync-up if you dont agree.
https://www.sisense.com/blog/better-sql-schema/

File: src/data/postgres/tables/device.test.ts (hunk 3 above)
Comment (vinayg-intel) [reply src/data/postgres/tables/device.test.ts]: updated

**Review by rsdmike — APPROVED** [review]

**Pull request merged**